README.md

```
@@ -1,591 +1 @@
-We welcome contributions! See [CONTRIBUTING.md](CONTRIBUTING.md) for guidelines.
-
-## License
-
-[Apache License 2.0](LICENSE)
-
-> **Note:** Versions prior to 2.0 are licensed under the [Mozilla Public License 2.0](https://www.mozilla.org/MPL/2.0/).
-
----
-
-**Found this useful?** Give us a star to help others discover OpenDataLoader.
+Added new feature
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)